src/6_branch.js

```diff
@@ -409,7 +409,9 @@ Branch.prototype['init'] = wrap(
 					"screen_width": screen.width || -1,
 					"screen_height": screen.height || -1,
 					"og_data": utils.openGraphDataAsObject(),
-					"hosted_deeplink_data": utils.scrapeHostedDeepLinkData()
+					"hosted_deeplink_data": utils.scrapeHostedDeepLinkData(),
+					"title": utils.scrapeTitle(),
+					"description": utils.scrapeDescription()
```

Comment:
scrapeXXX might be a good method name since websdk is open source I assume. Don't want to trigger an alarm to partners. getXXX() might be better.